src/client/Client.cc

```diff
@@ -8871,7 +8871,7 @@ loff_t Client::_lseek(Fh *f, loff_t offset, int whence)
 {
   Inode *in = f->inode.get();
   int r;
-  loff_t pos = -1;
+  loff_t pos = f->pos;
```

Comment:
It may even be clearest to just not initialize this here and just have SEEK_CUR case do:

`pos = f->pos + offset;`

Also, while you're in here...the default case in the switch should probably not ceph_abort(). A -EINVAL return (and maybe a dout() message) would be better. No need to crash the program for that case. If you feel motivated to fix that, I wouldn't complain.

Reply:
Your suggestion is very good, I have updated according to your suggestion. Please review, thanks.